code/modules/urist/items/clothes/uristclothes.dm

```diff
@@ -1813,3 +1813,16 @@ Update 26/07/2014 - All generic clothing goes under obj/item/clothing/under/uris
 		bio = ARMOR_BIO_SMALL,
 		rad = ARMOR_RAD_MINOR
 		)
+
+/obj/item/clothing/under/contortionist
+	name = "contortionist's jumpsuit"
+	desc = "A light jumpsuit useful for squeezing through narrow vents."
+	icon_state = "robotics2"
+	item_state = "robotics2"
+
+/obj/item/clothing/under/contortionist/proc/check_clothing(mob/user as mob)
```

Comment:
This proc name is not terribly informative. At a glance, I wouldn't realize this is even a custom proc defined only for this subclass. 

ventcrawl_check_clothing()? ventcrawling_allowed()?